tests/test_repository_manifest.py

```
@@ -0,0 +1,74 @@
+            )
+            package_metadata = package.stat()
+            database_metadata = database.lstat()
+
+            result = subprocess.run(
+                [sys.executable, str(REPOSITORY_MANIFEST), str(repository)],
+                cwd=REPO_ROOT,
+                text=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                check=True,
+            )
```

Ruff (0.16.1)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)